Gruntfile.js

```diff
@@ -389,58 +454,42 @@ module.exports = function (grunt) {
-                            "components/**/*.html",
-                            "components/**/assets/**/*",
-                            ".htaccess",
-                            "assets/**/*",
-                            "assets/images/{,*/}*.{webp}",
-                            "assets/fonts/**/*",
-                            "assets/sounds/**/*",
-                            "app/**/!(*.tpl).html",
-                            "index.html"
-                        ]
-                    }, {
-                        expand: true,
-                        cwd: "<%= yeoman.tmp %>/images",
-                        dest: "<%= yeoman.dist %>/client/assets/images",
-                        src: ["generated/*"]
-                    }, {
-                        expand: true,
-                        dest: "<%= yeoman.dist %>",
-                        src: [
-                            "package.json",
-                            "<%= yeoman.server %>/**/*"
-                        ]
-                    }, {
-                        expand : true,
-                        cwd: "<%= yeoman.tmp %>",
-                        dest: "<%= yeoman.dist %>/client",
-                        src : ["**/translations/*.json"]
-                    }, {
-                        cwd: "<%= yeoman.client %>",
-                        expand: true,
-                        dest: "<%= yeoman.dist %>/client",
-                        src: "usertests/**/*"
-                    }, {
-                        expand: true,
-                        cwd: "<%= yeoman.client %>",
-                        dest: "<%= yeoman.dist %>/client",
-                        src: [
-                            "bower_components/**/dist/**/Messages_*.json"
-                        ]
-                    }
-                ]
+                files: [{
+                    expand: true,
+                    dot: true,
+                    cwd: "<%= yeoman.client %>",
+                    dest: "<%= yeoman.dist %>/client",
+                    src: copyFiles
+                }, {
+                    expand: true,
+                    cwd: "<%= yeoman.tmp %>/images",
+                    dest: "<%= yeoman.dist %>/client/assets/images",
+                    src: ["generated/*"]
+                }, {
+                    expand: true,
+                    dest: "<%= yeoman.dist %>",
+                    src: [
+                        "package.json",
+                        "<%= yeoman.server %>/**/*"
+                    ]
+                }, {
+                    expand : true,
+                    cwd: "<%= yeoman.tmp %>",
+                    dest: "<%= yeoman.dist %>/client",
+                    src : ["**/translations/*.json"]
+                }, {
+                    cwd: "<%= yeoman.client %>",
+                    expand: true,
+                    dest: "<%= yeoman.dist %>/client",
+                    src: "usertests/**/*"
```

Comment:
Maybe we can remove this object right? Because doesn't exist any folders `usertests`.